certified-connectors/ZohoSign/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1661 @@
+                            "format": "int32",
+                            "description": "Signing order"
+                          },
+                          "recipient_name": {
+                            "type": "string",
+                            "description": "Recipient name"
+                          },
+                          "action_status": {
+                            "type": "string",
+                            "description": "Action status"
+                          },
+                          "recipient_countrycode": {
+                            "type": "string",
+                            "description": "Recipient countrycode"
+                          },
+                          "ishost": {
+                            "type": "boolean",
+                            "description": "host usage status"
+                          },
+                          "language": {
+                            "type": "string",
+                            "description": "language"
+                          },
+                          "in_person_name": {
+                            "type": "string",
+                            "description": "In person name"
+                          },
+                          "in_person_email": {
+                            "type": "string",
+                            "description": "In person email"
+                          },
+                          "verification_code": {
+                            "type": "string",
+                            "description": "Verification code"
+                          }
+                        }
+                      },
+                      "description": "actions"
+                    },
+                    "attachment_size": {
+                      "type": "integer",
+                      "format": "int32",
+                      "description": "Size of the attachment"
+                    }
+                  },
+                  "description": "requests"
+                },
+                "message": {
+                  "type": "string",
+                  "description": "message"
+                },
+                "status": {
+                  "type": "string",
+                  "description": "status"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Get a document",
+        "description": "Get details of a particular document",
+        "operationId": "get-document",
+        "parameters": [
+          {
+            "name": "RequestId",
+            "in": "path",
+            "required": true,
+            "type": "integer",
+            "format": "int64",
+            "x-ms-url-encoding": "single",
+            "description": "Request ID",
+            "x-ms-summary": "Request ID"
+          }
+        ]
+      },
+      "put": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
```

Comment:
@ZohoSign-integ Empty response schema is not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide a valid response schema definition for all the operations in the swagger. **This applies to all the empty schemas in the swagger.**